concepticondata/conceptrelations.tsv

```diff
@@ -770,3 +770,12 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3805	SHY OR ASHAMED	narrower	1894	ASHAMED
 3809	SCREAM (PRODUCE A CRY)	producedby	2522	SCREAM (NOISE)
 298	SHRIEK	similar	3809	SCREAM (PRODUCE A CRY)
+3213	HIRE OR RENT	narrower	3831	RENT (SOMETHING)
+947	STREAM	narrower	3821	BROOK
+1208	CAT	narrower	3825	MALE CAT
+1936	DEER	instanceof	3826	ROE DEER
+832	BEAN	instanceof	3839	BROAD BEAN
+3830	WALL	narrower	933	WALL (OF HOUSE)
```

Comment:
Is it `narrower` or `instance of`?

Reply:
Here I wasn't sure about the distinction between those relations either.

Reply:
I guess narrower because a Mauer and a Wand, while having many similarities, are in some sense an artificial grouping of two different things. But I could argue either way.